### PR TITLE
fasole: init at 1.2.3

### PR DESCRIPTION
**Pull request by ProggerX**
## Description of changes
Add fasole - minimalist's todo-list with md file format

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/fa/fasole/package.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  vendorHash = "sha256-V5jqsNy4Pu1AKikIZqEXERdggwBe3gXKMJVmgivVT6A=";
+
+  meta = with lib; {
+    description = "Minimalist's todo-list";
+    homepage = "https://github.com/ProggerX/fasole";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ proggerx ];
+    mainProgram = "fasole";
```
Comment (AndersonTorres) [inline pkgs/by-name/fa/fasole/package.nix:21]:
```suggestion
    license = lib.licenses.gpl3Only;
    mainProgram = "fasole";
    maintainers = with lib.maintainers; [ proggerx ];
```

File: pkgs/by-name/fa/fasole/package.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  src = fetchFromGitHub {
+    owner = "ProggerX";
+    repo = "fasole";
+    rev = "v${version}";
+    hash = "sha256-qcCJgz/YXfd8+9ST1U4YFxLLd25D8HrfZzsDGpKgCdM=";
+  };
+
+  vendorHash = "sha256-V5jqsNy4Pu1AKikIZqEXERdggwBe3gXKMJVmgivVT6A=";
+
+  meta = with lib; {
```
Comment (AndersonTorres) [inline pkgs/by-name/fa/fasole/package.nix:16]:
```suggestion
  meta = {
```

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -16248,6 +16248,11 @@
     githubId = 3153638;
     name = "Profpatsch";
   };
+  proggerx = {
+    email = "x@proggers.ru";
+    github = "ProggerX";
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:16253]: add githubId

**Comment by DontEatOreo** [comment]
Your commits are breaking [Commit Conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) please rebase.

In your case reword 2cc2f6cd6dd72451d53c00b35e6ec1584bda9ddd to `maintainers: add proggerx` and squash dee9d9cd47314141814d38b36b0c7fa1c0e7f202 into 71ab48f123d0053986281027b809b004a2bcbaad

**Comment by ProggerX** [comment]
@DontEatOreo is it better now?

**Comment by DontEatOreo** [comment]
Please change title to `fasole: init at 1.2.3`

**Comment by ProggerX** [comment]
oh f***

**Comment by ProggerX** [comment]
wait, i'll fix it

**Comment by ProggerX** [comment]
@DontEatOreo @AndersonTorres Is it correct now?

**Comment by DontEatOreo** [comment]
Result of `nixpkgs-review pr 309366` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fasole</li>
  </ul>
</details>

**Review by 4evy — APPROVED** [review]

**Comment by DontEatOreo** [comment]
Result of `nixpkgs-review pr 309366` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fasole</li>
  </ul>
</details>

**Review by AndersonTorres — APPROVED** [review]

**Comment by ProggerX** [comment]
Who can merge it?

**Comment by ProggerX** [comment]
yeeey

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**